src/ray/common/client_connection.cc

```diff
@@ -54,26 +55,6 @@ void SetFdCloseOnExec(const NativeHandleType &handle) {
   // https://learn.microsoft.com/en-us/windows/win32/sysinfo/handle-inheritance
   return;
 }
-#else
```

Comment:
let's keep the linux & windows implementations of `SetFdCloseOnExec` in the same place, so also move the `#if defined(_WIN32)` branch

Reply:
I removed the entire implementation of the Windows version of `SetFdCloseOnExec` and instead used macros to conditionally call it inside `client_connection.cc`. Since it does nothing and simply returns, and Windows does not use "file descriptors", I felt it was odd to keep it as a utility function.